pkg/apis/pravega/v1alpha1/pravega.go

```diff
@@ -332,10 +332,11 @@ type FileSystemSpec struct {
 
 // ECSSpec contains the connection details to a Dell EMC ECS system
 type ECSSpec struct {
-	ConfigUri   string `json:"configUri"`
-	Bucket      string `json:"bucket"`
-	Prefix      string `json:"prefix"`
-	Credentials string `json:"credentials"`
+	ConfigUri    string `json:"configUri"`
+	Bucket       string `json:"bucket"`
+	Prefix       string `json:"prefix"`
+	Credentials  string `json:"credentials"`
+	Certificates string `json:"certificates"`
```

Comment:
Could Credentials and Certificates be provided using the same secret object?

Reply:
This command is used to upload the certificate into k8s secret:
kubectl create secret generic ecs-certificate --from-file ./ecs-certificate.pem
This is a bit different to how we create ecs credentials using ecs-credentials.yaml, because the yaml could contain user name and password, but it is not convenient to copy entire PEM file content into the yaml.
How k8s admin generates secrets is probably out the concern of operator. All we care is the secret name, which in the case are "ecs-credentials" and "ecs-certificate" respectively.

Reply:
My question was more around, do we need 2 separate fields - **credentials** & **certificates** or would both be typically mounted using the same secret in the same location? In that case only 1 field should suffice